File: modulo1/segmento3/apunte3/ejercicio4/ejercicio4.java
import java.util.Scanner;
public class ejercicio4
{
	public static void main(String args[]){

	Scanner teclado = new Scanner(System.in);

	System.out.println("Ingrese un numero romano: ");
	String number = teclado.nextLine();

	String respuesta = "";

	if (number.toUpperCase().equals("I")){
		respuesta = "1";
	}else if (number.toUpperCase().equals("V")){
		respuesta = "5";

	}else if (number.toUpperCase().equals("X")){
		respuesta = "10";
	}else if (number.toUpperCase().equals("L")){
		respuesta = "50";
	}else if (number.toUpperCase().equals("C")){
		respuesta = "100";
	}else{
		respuesta = "Numero Invalido";
	}
		System.out.println(respuesta);
	}
}

File: modulo1/segmento3/apunte3 "IF - ELSE IF (ELSE)"/ejercicio4/ejercicio4.py
letra = input("Ingrese un numero romano: ");

if (letra == "I"):
	numero = 1

elif (letra == "V"):
	numero = 5

elif (letra == "X"):
	numero = 10

elif (letra == "M"):
	numero = 50

elif (letra == "C"):
	numero = 100

elif (letra == "M"):
	numero = 1000

else:
	numero("Numero romano no valido")

print("El numero romano ingresado es: " + str(numero))
